src/entities/user/model/user.slice.ts

```diff
@@ -0,0 +1,29 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit"
+
+export interface UserStore {
+  id: string | number | null
+  email: string | null
+  fullName: string | null
+}
+
+const initialState: UserStore = {
+  email: null,
+  fullName: null,
+  id: null,
+}
+
+const userSlice = createSlice({
+  name: "user",
+  initialState,
+  reducers: {
+    save: (state, action: PayloadAction<UserStore>) => {
+      return { ...state, ...action.payload }
```

Comment:
Redux Toolkit uses Immer internally
https://redux-toolkit.js.org/usage/immer-reducers

please make use of it when writing reducers.

Reply:
Done